src/Streams.ts

```diff
@@ -0,0 +1,337 @@
+import { Buffer } from 'buffer'
```

Comment:
Use https://github.com/feross/buffer instead of Node's buffer. And we need to ensure we are getting the buffer from the library.

Reply:
Done, although I'm not sure how to ensure its using the buffer from the library. I have put this line into webpack config:
```
module.exports = {
  ...
  Buffer: require('buffer'),
  ...
}
```

Reply:
this is now working